pkg/cluster/EphemeralConatinersService.go

```diff
@@ -0,0 +1,160 @@
+package cluster
+
+import (
+	"errors"
+	"github.com/devtron-labs/devtron/pkg/cluster/repository"
+	"github.com/go-pg/pg"
+	"go.uber.org/zap"
+	"time"
+)
+
+type EphemeralContainerRequest struct {
+	BasicData    *EphemeralContainerBasicData    `json:"basicData"`
+	AdvancedData *EphemeralContainerAdvancedData `json:"advancedData"`
+	Namespace    string                          `json:"namespace"`
+	ClusterId    int                             `json:"clusterId"`
+	PodName      string                          `json:"podName"`
+	UserId       int32                           `json:"-"`
+}
+
+type EphemeralContainerAdvancedData struct {
+	Manifest string `json:"manifest"`
+}
+
+type EphemeralContainerBasicData struct {
+	ContainerName       string `json:"containerName"`
+	TargetContainerName string `json:"targetContainerName"`
+	Image               string `json:"image"`
+}
+
+type EphemeralContainerService interface {
+	SaveEphemeralContainer(model EphemeralContainerRequest) error
```

Comment:
i think the name should be ``CreateEphemeralContainer``

Reply:
I had suggeted the name of func ``SaveEphemeralContainer`` to be ``CreateEphemeralContainer`` not service name